ci/tests/general/general_vpool.py

```diff
@@ -546,3 +546,19 @@ def unmount_vpool(vpool, root_client):
         mountpoint = '/mnt/{0}'.format(vpool.name)
         if mountpoint in General.get_mountpoints(root_client):
             root_client.run('umount {0}'.format(mountpoint))
+
+    @staticmethod
+    def get_vpool_storage_routers(vpool):
```

Comment:
Don't know whether this is useful, a vpool has a set amount of storagedrivers and each storagedriver is always linked to 1 storagerouter

So this method can be replaced to a 1-liner in code
[sd.storagerouter for sd in vpool.storagedrivers]